Firestore/core/test/firebase/firestore/model/blob_test.cc

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/firebase/firestore/model/blob.h"
+
+#include "gtest/gtest.h"
+
+namespace firebase {
+namespace firestore {
+namespace model {
+
+TEST(Blob, Getter) {
+  Blob a = Blob::CopyFrom(reinterpret_cast<const uint8_t*>("\1\2\3"), 3);
```

Comment:
nit: It would help the readability of these tests if you made little functions to hide these casts. 

Something like

```c++
const uint8_t* Bytes(const char* value) { ... }
uint8_t* Bytes(char* value) { ... }
```

And perhaps the other direction too.